### DISCUSSION TITLE
Value reset to `defaultvalue` on mount.

### DISCUSSION THREAD

**Discussion by behzadmehrabi** [react-hook-form/react-hook-form#5754] [Q&A] [answered]
Hi,
I'm using react-window to virtualize my table of inputs, I'm using `Controller` component. when I change the input value it works nicely, but when I scroll down the table and head back up again(so the input will be unmounted and mounted again), the input value has been reset to the `defaultValue`. what I am I doing wrong?

I'm using FieldArray.
`shouldUnregister` is set to `false`.

**Comment by bluebill1049** [comment, marked as answer]
can you provide a simple reproducable csb?

**Reply by behzadmehrabi** [reply]
https://codesandbox.io/s/beautiful-rubin-2b8vg?file=/src/App.tsx

**Reply by bluebill1049** [reply]
https://react-hook-form.com/advanced-usage#Workingwithvirtualizedlists?
